pi: add page (#4653)
tldr-pages/tldr: pages/linux/pi.md

# pi

> > Prints 100 decimal digits of Archimedes' constant pi or a number of digits specified by an integer parameter on the command line.

Reviewer: The command could be used for Math operations in Bash, so it's something more than display numbers:
```suggestion
> Compute decimal Archimedes' constant Pi on the command line.
```

> More information: <http://manpages.ubuntu.com/manpages/trusty/man1/pi.1.html>.

> - Print 100 decimal digits of Archimedes' constant pi:

Reviewer: ```suggestion
- Display 100 decimal digits of Archimedes' constant Pi:
```

`pi`

> - Display `number` decimal digits of Archimedes' constant Pi:

Reviewer 2: Perhaps this will read better?
```suggestion
- Display a specified number of decimal digits of Archimedes' constant Pi:
```

> - Print *number* decimal digits of Archimedes' constant pi:

Reviewer: ```suggestion
- Display `number` decimal digits of Archimedes' constant Pi:
```

`pi {{number}}`

> - Show summary of options:

Reviewer: ```suggestion
- Display help:
```

`pi --help`

> - Show current version of program:

Reviewer: ```suggestion
- Display version:
```

`pi --version`

> - Output recommended readings:

Reviewer: ```suggestion
- Display recommended readings:
```
